cmd/loadTests.go

```diff
@@ -961,17 +1000,75 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 				continue
 			}
 
-			/*
-				This part adds the Integration test scenario part
-				It's considered also as part of the resources creation since Integration test scenarios are resources as well
-			*/
 			var integrationTestScenario *integrationv1beta1.IntegrationTestScenario
+
+			startTimeForIts := time.Now()
 			integrationTestScenario, err = framework.AsKubeDeveloper.IntegrationController.CreateIntegrationTestScenario_beta1(ApplicationName, usernamespace, testScenarioGitURL, testScenarioRevision, testScenarioPathInRepo)
+			itsCreationTime := time.Since(startTimeForIts)
```

Comment:
Shall we measure the time from creation to `IntegrationTestScenarioValid` condition being true below?

Reply:
Yes Jan
If you look beneath the following line
"if condition.Type == "IntegrationTestScenarioValid" && condition.Status == "True" {"
It calculates the itsActualCreationTime value and assigns it the metricsConstants.MetricActualItsCreationTimeGauge
This part is waiting for a new PR on github.com/redhat-appstudio-qe/perf-monitoring to add this MetricActualItsCreationTimeGauge metric as the other metrics related to its

Reply:
Actually noticed today that the integration test scenarios metrics are already available in github.com/redhat-appstudio-qe/perf-monitoring and I added these metrics handling to the code

Reply:
The usage of the actual metrics in the monitoring repo
if condition.Type == "IntegrationTestScenarioValid" && condition.Status == "True" {
	lastTransitionTime := condition.LastTransitionTime.Time
	itsActualCreationTime := lastTransitionTime.Sub(startTimeForIts)
	MetricsWrapper(MetricsController, metricsConstants.CollectorIntegrationTestsSC, metricsConstants.MetricTypeGuage, metricsConstants.MetricActualIntegrationTestSenarioCreationTimeGauge, itsActualCreationTime.Seconds())

Reply:
Hello @sawood14012 . Could you please confirm this is measured correctly?

Reply:
i think it should be the creationTimestamp not the LastTransitionTime

Reply:
Sorry my bad i thought something else the Last trainsition time seems to do the trick here